apps/mongodbnotify/manifest.yaml

```diff
@@ -0,0 +1,6 @@
+---
+id: mongodb-notify
+name: MongoDB Notify
+summary: Message notifications
+desc: Uses MongoDB for notifications. Can be used with Home Assistant to get Notifications to Tidbyt.
```

Comment:
Could you please update the app description a bit? Would be good if it explained that users will likely have to go read the README here in the community repo to get this set up.

Reply:
Hi, @matslina,
I've updated the desc as you requested.